galasa-ui/src/components/Footer.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+"use client";
+
+import React, { useEffect, useState } from 'react';
+import styles from "@/styles/Footer.module.css";
+import { Theme } from '@carbon/react';
+
+interface FooterProps {
+  serviceHealthyPromise: Promise<boolean>;
+  clientVersionPromise?: Promise<string | undefined>;
+}
+
+const Footer = ({ serviceHealthyPromise, clientVersionPromise }: FooterProps) => {
+
+  const [isHealthOk, setIsHealthOk] = useState(true);
+  const [apiVersion, setApiVersion] = useState("");
+
+  const checkServiceHealth = async () => {
+    const isPingSuccessful = await serviceHealthyPromise;
+    setIsHealthOk(isPingSuccessful);
+  };
+
+  const getApiVersion = async () => {
+    const apiVersion = await clientVersionPromise;
+    if(apiVersion) {
+      setApiVersion(apiVersion);
+    }
+  };
+
+  useEffect(() => {
+
+    checkServiceHealth();
+    getApiVersion();
+
+  }, []);
```

Comment:
A warning will appear here about useEffect missing the promises as dependencies, can this be addressed? Might need some investigation to make sure we're not constantly re-rendering the footer